lib/presentation/shared_widgets/expandable_text.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+
+import '../themes/constants.dart';
+
+/// Text widget to show read more and read less
+class ExpandableText extends StatefulWidget {
+  const ExpandableText(
+    this.text, {
+    Key? key,
+    this.trimLines = 3,
+    this.clickableTextColor = kAccentColor,
+    this.readMoreText = "... read more",
```

Comment:
maybe just "more" and "less"?

Reply:
Also the three dots (ellipsis) should not be the same color as the more text right?

Reply:
I was following the design and at least that's what it specifies.

<img width="1025" alt="Bildschirmfoto 2022-02-11 um 15 56 06" src="https://user-images.githubusercontent.com/44906984/153614321-b27eb26e-e585-48ec-b494-59baa50bce3c.png">

_Originally posted by @laschnik in https://github.com/CollActionteam/collaction_app/issues/190#issuecomment-1036297382_

Reply:
Ah yes I see, I like the changes you just committed better haha